hub/policy/src/main/java/uk/gov/ida/hub/policy/domain/controller/CountrySelectedStateController.java

```diff
@@ -100,60 +83,135 @@ public AuthnRequestFromHub getRequestFromHub() {
         return requestToSendFromHub;
     }
 
-    // TODO: Add to check if state has matching service entity id
-    public String getMatchingServiceEntityId() {
-        return transactionsConfigProxy.getMatchingServiceEntityId(state.getRequestIssuerEntityId());
-    }
+    public EidasAttributeQueryRequestDto getEidasAttributeQueryRequestDto(InboundResponseFromCountry translatedResponse) {
```

Comment:
Moving this from `AuthnResponseFromCountryService` seems like a redundant move. The test class is already stubbing out the service so it's not a good reason really...

I'm weary of moving thing around without well justified reasons because thing are likely to be at where they were for good reasons. Moreover, this is making the diff absolutely huge and very hard to figure out what the real code changes are.